### PR TITLE
Force to compile with UTF8 encoding

### PR DESCRIPTION
**Pull request by chenjunjiedada**

### CODE REVIEW

**Comment by rdblue** [comment]
What problem does this fix? Isn't UTF-8 the default?

**Comment by chenjunjiedada** [comment]
Yes, according to doc [here](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.compile.CompileOptions.html), the default encoding is set to null, which uses the system default encoding.

File: build.gradle [hunk 1]
```diff
@@ -64,6 +64,9 @@ subprojects {
   apply plugin: 'maven' // make pom files for deployment
   apply plugin: 'nebula.maven-base-publish'
 
+  compileJava {options.encoding = "UTF-8"}
```
Comment (rdblue) [inline build.gradle]: These blocks should use newlines and match the formatting of the rest of the file.

Comment (chenjunjiedada) [reply build.gradle]: I see, updated in latest commit

**Comment by rdblue** [comment]
Fixed. Thanks @chenjunjiedada!

**Pull request merged**